hyperion/app/creation_steps/step8_review_and_refine/consistency_check/models.py

```
@@ -0,0 +1,172 @@
+from pydantic import BaseModel, Field
+
+from app.grpc.hyperion_pb2 import ArtifactLocation, ArtifactType, ConsistencyIssue, ConsistencyIssueCategory, ConsistencyIssueSeverity, ConsistencyIssueType
+
+
+class SeverityEnum(str, Enum):
+    LOW = "LOW"
+    MEDIUM = "MEDIUM"
+    HIGH = "HIGH"
+
+class StructuralConsistencyIssueCategory(str, Enum):
+  METHOD_SIGNATURE_MISMATCH = "METHOD_SIGNATURE_MISMATCH";                 # Method return type, parameters, or visibility differs between artifacts
```

Ruff (0.11.9)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)